napari/layers/image/_image_constants.py

```diff
@@ -88,3 +89,23 @@ class ImageRendering(StringEnum):
     MINIP = auto()
     ATTENUATED_MIP = auto()
     AVERAGE = auto()
+
+
+class VolumeDepiction(StringEnum):
+    """Depiction: 3D depiction mode for images.
+
+    Selects a preset depiction mode in vispy
+            * volume: images are rendered as 3D volumes.
+            * plane: images are rendered as 2D planes embedded in 3D.
+    """
+
+    VOLUME = auto()
+    PLANE = auto()
+
+
+VOLUME_DEPICTION_TRANSLATION = dict(
```

Comment:
I'd say that now that dicts are insertion-ordered by default, it *probably* won't matter, but nevertheless, it would be cheap to change this to using OrderedDict.